cmd/exec.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"context"
+	"github.com/urfave/cli/v2"
+	"gopkg.in/nullstone-io/go-api-client.v0"
+	"gopkg.in/nullstone-io/nullstone.v0/app"
+)
+
+var Exec = func(providers app.Providers) *cli.Command {
+	return &cli.Command{
+		Name:      "exec",
+		Usage:     "Execute command on running service",
```

Comment:
In the usage description, might want to mention that by not providing a [command], you will connect to a shell session in one of the containers.

Reply:
Done.